otter/log/formatters.py

```diff
@@ -261,6 +271,9 @@ def error_formatting_observer(event):
                 message = repr(excp)
                 event['traceback'] = event['failure'].getTraceback()
                 event['exception_type'] = excp.__class__.__name__
+                details = serialize_to_jsonable(excp)
+                if details != repr(excp):
```

Comment:
This is because exception repr is already stored in message. I was thinking of not duplicating it in `error_details`. The intention is the give extra details only when it is there. I don't feel that strongly about it. If you think its ok to put same info, I can change it.